lib/FileUtils

```diff
@@ -0,0 +1,33 @@
+/**
+ * Created by AsherS on 8/20/15.
+ */
+
+var fs = require('fs');
+var Q = require('q');
+
+var FileUtils = (function(){
+
+    function FileUtils(){
+
+    }
+
+    FileUtils.listDirFiles = function(path) {
```

Comment:
I'm pretty sure q-io.listTree [https://github.com/kriskowal/q-io] does exactly this